### ISSUE TITLE
Article template

### ISSUE THREAD

**Issue by robsongajunior** [es6rocks/harmonic-theme-jsrocks#4]
more posts template is not use nunjucks template, it's hard code mode.

**Comment by robsongajunior** [comment]
At moment, is not a good idea. We are using a small template method at JSROCKS class!

**Comment by UltCombo** [comment]
Yeah, it would be nice to move it to the template html if possible.

**Comment by robsongajunior** [comment]
yeaah, i'ts possible. i will take a look too :p
